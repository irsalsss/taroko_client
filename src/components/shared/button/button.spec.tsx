import Button from "./button";
import { render } from "@/utils/test/react-testing-setup";

describe("Button", () => {
  it("should render button successfully", () => {
    const { baseElement } = render(<Button label='Click me' />);

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
