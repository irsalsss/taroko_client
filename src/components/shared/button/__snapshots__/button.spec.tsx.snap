// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button successfully 1`] = `
<body>
  <div>
    <button
      class="button"
      label="Click me"
      role="button"
    >
      Click me
    </button>
  </div>
</body>
`;
